src/pages/VenuePage/VenuePage.tsx

```diff
@@ -211,10 +154,10 @@ const VenuePage: React.FC = () => {
   // @debt refactor this + related code so as not to rely on using a shadowed 'storeAs' key
   //   this should be something like `storeAs: "currentUserPrivateChats"` or similar
   useFirestoreConnect(
-    user?.uid
+    userId
       ? {
           collection: "privatechats",
```

Comment:
This should be refactored into a `usePrivateChats` or similar hook.

Reply:
We have an issue for that. It's in progress and is related to chats https://github.com/sparkletown/internal-sparkle-issues/issues/109

Reply:
Awesome. Sounds good.